src/content/en/tools/workbox/examples.md

```diff
@@ -0,0 +1,55 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Workbox Examples.
+
+{# wf_published_on: 2015-01-01 #}
+{# wf_updated_on: 2017-05-16 #}
+
+# Examples {: .page-title }
+
+This page contains examples for some of the Workbox modules.
+
+Workbox is a set of small modules that can be used on their own, but we've
+also combined the most common pieces into one module, `workbox-sw`. This module
+supports precaching, routing of fetch events, runtime caching and more.
+
+## workbox-sw
+
+The `workbox-sw` example demonstrates a complete service worker
+implementation, demonstrating the common parts of the API. It's an in-depth
+exploration of everything you'd need to build a production-ready service worker.
+
+<a href="https://workbox-samples.glitch.me/examples/workbox-sw/" class="btn">View Example</a>
```

Comment:
We should include a note (in the Readme/Contributing/elsewhere) that shares how these examples are maintained or updated. e.g are they auto-pushed to Glitch? is there a point-of-contact for getting them updated if we notice a bug etc.

Reply:
Do you want it here or on the demos?

I think you are worrying about stuff before it's happened.